Class-7-(Interfaces).md

```diff
@@ -0,0 +1,19 @@
+# Class 7: Interfaces and Polymorphism
+
+## In the prep work this week, the students learned
+1. What polymorphism is.
+1. What interfaces are and the difference between interfaces and abstract classes.
+1. Different interfaces that are a part of Java and how they can be used in our code.
+1. Best practices around unit tests and interfaces.
+
+## Announcements
+1. Check with your course manager for any important announcements.
```

Comment:
How about a reminder that assignment 2 is due before the next class?

Reply:
Done! Thanks!